zerver/actions/create_realm.py

```diff
@@ -254,25 +259,33 @@ def do_create_realm(
         # We use acting_user=None for setting the initial plan type.
         do_change_realm_plan_type(realm, Realm.PLAN_TYPE_LIMITED, acting_user=None)
 
-    admin_realm = get_realm(settings.SYSTEM_BOT_REALM)
-    sender = get_system_bot(settings.NOTIFICATION_BOT, admin_realm.id)
-    # Send a notification to the admin realm
-    signup_message = _("Signups enabled")
+    # Send a notification to the admin realm when a new organization registers.
+    if settings.CORPORATE_ENABLED:
+        admin_realm = get_realm(settings.SYSTEM_BOT_REALM)
+        sender = get_system_bot(settings.NOTIFICATION_BOT, admin_realm.id)
 
-    try:
-        signups_stream = get_signups_stream(admin_realm)
-        topic = realm.display_subdomain
+        support_url = get_support_url(realm)
+        organization_type = get_org_type_display_name(realm.org_type)
 
-        internal_send_stream_message(
-            sender,
-            signups_stream,
-            topic,
-            signup_message,
-        )
-    except Stream.DoesNotExist:  # nocoverage
-        # If the signups stream hasn't been created in the admin
-        # realm, don't auto-create it to send to it; just do nothing.
-        pass
+        with override_language(admin_realm.default_language):
+            message = _("[{realm_name}]({support_link}). Organization type: {type}").format(
+                realm_name=realm.display_subdomain, support_link=support_url, type=organization_type
+            )
+            topic = _("new organization registrations")
```

Comment:
@alya would something like "New organizations" feel nicer here?

Reply:
Sur, that works.